docs/UserGuide.md

```diff
@@ -153,12 +143,16 @@ Example:
 
 Adds an existing tag to an existing contact.
 
-Format: `tag add n/NAME t/TAG`
+Format: `tag add INDEX t/TAG`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A person can have any number of tags. Add as many as you want.
+</div>
 
-* `NAME` is the name of the contact to add `TAG` to.
+* Adds a `TAG` to the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 
 Example:
-* `tag add n/John Doe t/friend` adds the friend tag to John Doe.
+* `tag add 1 t/friend` adds the friend tag to John Doe.
```

Comment:
Specify that John Doe is index 1 on the currently displayed list? Specify that the `friend` tag already exists?

Reply:
Ok. I made changes.